README.md

```diff
@@ -3,9 +3,9 @@
 
 Salt formula provisioning a Slurm cluster
 
-Availables states:
-  * Munge
-  * Screen
-  * Slurm
-  * Slurm Database
-  * SSH
+To install Slurm nodes, you need to copy (on Slurm mater node)
```

Comment:
Maybe move that lower in an "Instructions" subsection instead of replacing what the formula contains?